README.md

```diff
@@ -1,5 +1,9 @@
 # Open Source Help Community
 
+ [![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
+
+
```

Comment:
No need of so many extra line breaks. One line break between the badge and the logo should do. :)